daemon/mgr/container.go

```diff
@@ -1290,3 +1348,49 @@ func checkBind(b string) ([]string, error) {
 
 	return arr, nil
 }
+
+func parseBindMode(mp *types.MountPoint, mode string) error {
```

Comment:
I am afraid that we need unit test for this function. 😄

Reply:
ok, I add it.

Reply:
I have added unit tests. PTAL.